src/test/java/gov/healthit/chpl/aqa/stepDefinitions/CollectionsPagesSteps.java

```diff
@@ -63,4 +63,25 @@ public void verifyCMSFAQLink(final String cmsfaqLink) {
         assertTrue(link.contains(cmsfaqLink), "Expect " + cmsfaqLink + " to be found in " + link);
     }
 
+    /**
+     * Get user to the Collections page.
+     */
+    @Given("^I am on the Collections page$")
+    public void iAmOnCollectionsPage() {
```

Comment:
This step is a less useful one than the one in `loadCollectionsPage`, and unclear. What "collections" page? There are five or six of them, and this one loads a single one without being clear that it's doing that

Reply:
There is an existing step in this file (see line 50) that can be used in new scenario as - 'Given I am on "SED Information for 2015 Edition" collections page: "sed"', no need for new step and step definition.